library/src/main/java/com/venmo/view/ArrowAlignment.java

```diff
@@ -0,0 +1,14 @@
+package com.venmo.view;
+
+public enum ArrowAlignment {
+    LEFT, CENTER, RIGHT, CUSTOM;
+
+    public static ArrowAlignment getAlignment(int ordinal) {
```

Comment:
Using the ordinal is a little dangerous since you then need to keep everything in the same order. I think creating a variable for each one and assigning

```
private static final int LEFT_VALUE = 1;
LEFT(LEFT_VALUE), CENTER(CENTER_VALUE)
```

etc is safer.